content/02.main-text.md

```diff
@@ -62,6 +110,8 @@ First, scientific precedence can now be indisputably established.
 Second, timestamps can protect against attempts to rewrite a manuscript's history.
 Since timestamps cannot be backdated, alternative histories would have to be created in advance, which is generally infeasible.
 Therefore, timestamping can help ensure accurate manuscript histories, potentially alleviating certain authorship disputes.
+Note that OpenTimestamps is highly scalable and free to use since it encodes many timestamps into a single Bitcoin transaction.
```

Comment:
What should we do when we encounter a technically important point that would be prohibitive to explain to a layperson?

Reply:
Since I know a lot less about blockchain/bitcoin than @dhimmel, I am going to create a new issue and I will try to rewrite this paragraph such that it is easy for me to understand. That should be a reasonable first step towards a general audience. 👍